config/system.yaml

```diff
@@ -35,9 +35,7 @@ errors:
   log: true
 
 debugger:
-  enabled: false
-  twig: false
+  enabled: true
+  twig: true
```

Comment:
We don't want to turn debugging on in production. @emb03 please remove all the above configs.